picard/util/_astrcmp.c

```diff
@@ -156,8 +156,8 @@ astrcmp(PyObject *self, PyObject *args)
 
 	us1 = PyUnicode_AS_UNICODE(s1);
 	us2 = PyUnicode_AS_UNICODE(s2);
```

Comment:
Isn't this also deprecated ?
https://docs.python.org/3/c-api/unicode.html#c.PyUnicode_AS_UNICODE

Reply:
Indeed it is. There was just no warning output, probably because these are macros. I will update the PR